hw3/index31.js

```diff
@@ -0,0 +1,18 @@
+const numbersList = [1, 2, 3, 4];
+
+const changeNumbers = (numbers) => {
+  if (!Array.isArray(numbers)) {
+    return;
+  }
+
+  return numbers.map((number) => {
+    if (number % 2 === 0) {
+        return number + 1;
+    }  
+    
+    return number - 1;
```

Comment:
here better to do it via ternary  (number) => `number % 2 === 0 ? number + 1 : number - 1`